spring-cloud-commons/src/main/java/org/springframework/cloud/client/serviceregistry/ServiceRegistry.java

```diff
@@ -0,0 +1,10 @@
+package org.springframework.cloud.client.serviceregistry;
+
+/**
+ * @author Spencer Gibb
+ */
+public interface ServiceRegistry<R extends Registration> {
```

Comment:
here or somewhere else, we should qualify with an example of what a registration might be